udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -25,10 +25,11 @@ function createCommentList(inputText) {
 function getCommentData() {
   fetch('/data')                          // sends a request to /data
       .then(response => response.json())  // parses the response as JSON
-      .then((myComments) => {  // now we can reference the fields in myComments!
+      .then((myComments) => {  // now we can reference the fields in myObject!
```

Comment:
As suggested in the previous PR, consider renaming myComments -> comments (to increase conciseness at no cost to clarity)